src/views/examples/Register.js

```diff
@@ -33,126 +35,114 @@ import {
+  const validateEmail = (e) => {
+    var email = e.target.value
+  
+    if (!validator.isEmail(email)) {
+      setEmailError('invalid!')
+    } else {
+      setEmailError('')
+    }
+  }
+  
+  return ( 
     <>
       <Col lg="6" md="8">
         <Card className="bg-secondary shadow border-0">
-          <CardHeader className="bg-transparent pb-5">
-            <div className="text-muted text-center mt-2 mb-4">
-              <small>Sign up with</small>
-            </div>
-            <div className="text-center">
-              <Button
-                className="btn-neutral btn-icon mr-4"
-                color="default"
-                href="#pablo"
-                onClick={(e) => e.preventDefault()}
-              >
-                <span className="btn-inner--icon">
-                  <img
-                    alt="..."
-                    src={
-                      require("../../assets/img/icons/common/github.svg")
-                        .default
-                    }
-                  />
-                </span>
-                <span className="btn-inner--text">Github</span>
-              </Button>
-              <Button
-                className="btn-neutral btn-icon"
-                color="default"
-                href="#pablo"
-                onClick={(e) => e.preventDefault()}
-              >
-                <span className="btn-inner--icon">
-                  <img
-                    alt="..."
-                    src={
-                      require("../../assets/img/icons/common/google.svg")
-                        .default
-                    }
-                  />
-                </span>
-                <span className="btn-inner--text">Google</span>
-              </Button>
-            </div>
-          </CardHeader>
           <CardBody className="px-lg-5 py-lg-5">
-            <div className="text-center text-muted mb-4">
-              <small>Or sign up with credentials</small>
-            </div>
-            <Form role="form">
+           
+            <Form role="form" onsubmit={()=> handlesubmit()}>
               <FormGroup>
                 <InputGroup className="input-group-alternative mb-3">
                   <InputGroupAddon addonType="prepend">
                     <InputGroupText>
-                      <i className="ni ni-hat-3" />
+                      <i className="ni ni-email-83" />
                     </InputGroupText>
                   </InputGroupAddon>
-                  <Input placeholder="Name" type="text" />
+                  <Input
+                    placeholder="IITK Email ID"
+                    type="email"
+                    autoComplete="off"
+                    onChange={(e) => validateEmail(e)}
+                  />
+                   <div className="text-muted text-center mt-2 mb-4">
+                       <span style={{
+                          fontWeight: 'light',
+                         color: 'red',
+                         }}>{emailError}
```

Comment:
Invalid is not looking good please change its position to next line or between email and password.